spec/operators/concat-spec.ts

```diff
@@ -2,210 +2,245 @@ import { expect } from 'chai';
 import { of, Observable } from 'rxjs';
 import { concat, mergeMap } from 'rxjs/operators';
 import { TestScheduler } from 'rxjs/testing';
-import { hot, cold, expectObservable, expectSubscriptions } from '../helpers/marble-testing';
+import { observableMatcher } from '../helpers/observableMatcher';
 
 declare function asDiagram(arg: string): Function;
 
 declare const rxTestScheduler: TestScheduler;
 
 /** @test {concat} */
 describe('concat operator', () => {
+  let testScheduler: TestScheduler;
+
+  beforeEach(() => {
+    testScheduler = new TestScheduler(observableMatcher);
+  });
+
   asDiagram('concat')('should concatenate two cold observables', () => {
-    const e1 =   cold('--a--b-|');
-    const e2 =   cold(       '--x---y--|');
-    const expected =  '--a--b---x---y--|';
+    testScheduler.run(({ cold, expectObservable }) => {
+      const e1 =   cold('--a--b-|');
+      const e2 =   cold(       '--x---y--|');
+      const expected =  '--a--b---x---y--|';
```

Comment:
When converting to `run`, we need to move the whitespace padding into the actual strings, so that we can use Prettier to format the source once the conversions are done. Prettier will collapse any whitespace outside of the string literals to a single space character and that will mess up the alignment.

Thanks for doing this, Zack.